ControlFlow/Program.cs

```diff
@@ -15,22 +12,47 @@ public static class ControlFlow
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
         {
-            return default;
+            if (number > 1 && number < 10)
+            {
+                return "Valid";
+            }
+            else
+            {
+                return "Invalid";
+            }
         }
 
         // 1.2 - Write a method which takes two integers and returns the maximum of the two. If they
         // are the same, return either one.
         public static int Maximum(int number1, int number2)
         {
-            return default;
+            if (number1 <= number2)
+            {
+                return number2;
+            }
+            else
+            {
+                return number1;
+            }
         }
 
         // 1.3 - Write a method which takes two integers, the width and height of an image. Then
         // return a string to tell if the image is "Landscape" or "Portrait". Return "Square" if
         // they are equal. Assume non-negative values.
         public static string LandscapeOrPortrait(int width, int height)
         {
-            return default;
+            if(width > height)
+            {
+            return "Landscape";
+            }
+            else if(height > width)
+            {
+            return "Portrait";
+            }
+            else
+            {
+            return "Square";
+            }
```

Comment:
I would recommend reducing the number of returns that you are using. You can make a variable that captures all your desired strings and just output that singular variable. Improves code readability this way.

```
var result = "Landscape";
result = "Portrait";
result = "Square";

return result;
```
something like this

Reply:
One thing I will add is to have interior code indented so it is easier to read. Without these indents it is difficult to tell what is inside the if statements and what is its own line of code. This is done to improve readability and is standard coding practice.
EX. using your code as shown
```
if(width > height)
{
     return "Landscape";
}
else if (height > width)
{
     return "Portrait";
}
... and so on
```